.eslintrc.js

```diff
@@ -63,6 +63,8 @@ module.exports = {
         '@typescript-eslint/no-shadow': ['error'],
         'no-throw-literal': ['error'],
         'no-trailing-spaces': ['error'],
+        'no-unneeded-ternary': ['error'],
+        '@typescript-eslint/no-unneeded-ternary': ['error'],
```

Comment:
This line isn't needed.

There are some rules that need to be specifically enabled for typescript and disabled in the default rules due to syntax differences in typescript vs javascript. This is not one of those so only the default rule needs enabled. 🙂

Reply:
so remove the 67th line?

Reply:
yup

Reply:
done